task/sql-tasks.js

```diff
@@ -177,7 +275,16 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+           (SELECT 
+               COUNT(ProductName) 
+            FROM Products)  as 'TotalOfCurrentProducts',
+           COUNT(ProductName) as 'TotalOfDiscontinuedProducts'
+        FROM Products
+        WHERE Discontinued != 0

```

Comment:
that's good, but try to re-write query like this

it's better to write this query like this

select
(select count()...) as 'T'
(select count()...) as 'TT'
notice that from clause is optional